projects/js-packages/publicize-components/src/components/connection-management/index.tsx

```diff
@@ -15,16 +15,13 @@ import styles from './style.module.scss';
 const ConnectionManagement = ( { className = null, disabled = false } ) => {
 	const { refresh } = useSocialMediaConnections();
 
-	const { connections, deletingConnections, updatingConnections } = useSelect( select => {
-		const { getConnections, getDeletingConnections, getUpdatingConnections } = select( store );
+	const connections = useSelect( select => select( store ).getConnections(), [] );
 
-		return {
-			connections: getConnections(),
-			deletingConnections: getDeletingConnections(),
-			updatingConnections: getUpdatingConnections(),
-		};
-	}, [] );
+	const deletingConnections = useSelect( select => select( store ).getDeletingConnections(), [] );
 
+	const updatingConnections = useSelect( select => select( store ).getUpdatingConnections(), [] );
```

Comment:
I am confused by this change.

According to the [blog post you linked](https://developer.wordpress.org/news/2024/03/how-to-work-effectively-with-the-useselect-hook/#do-all-selections-from-the-same-store-in-one-callback), we should

> Do all selections from the same store in one callback

Reply:
OK, it needs some clarification, as mentioned [here](https://developer.wordpress.org/news/2024/03/how-to-work-effectively-with-the-useselect-hook/#comment-17479).

Reply:
Yeah, sorry that comment is awaiting moderation. The point is that the result of such a `useSelect` should either have scalar values or objects with consistent reference.